sormas-backend/src/main/java/de/symeda/sormas/backend/campaign/CampaignFacadeEjb.java

```diff
@@ -357,38 +345,32 @@ public void deleteCampaign(String campaignUuid) {
 					+ I18nProperties.getString(Strings.entityCampaigns).toLowerCase() + ".");
 		}
 
-		campaignService.delete(campaignService.getByUuid(campaignUuid));
+		service.delete(service.getByUuid(campaignUuid));
 	}
 
 	@Override
-	public void archiveOrDearchiveCampaign(String campaignUuid, boolean archive) {
+	protected void pseudonymizeDto(Campaign source, CampaignDto dto, Pseudonymizer pseudonymizer) {
 
-		Campaign campaign = campaignService.getByUuid(campaignUuid);
-		campaign.setArchived(archive);
-		campaignService.ensurePersisted(campaign);
 	}
 
 	@Override
-	public CampaignReferenceDto getReferenceByUuid(String uuid) {
-		return toReferenceDto(campaignService.getByUuid(uuid));
+	protected void restorePseudonymizedDto(CampaignDto dto, CampaignDto existingDto, Campaign entity, Pseudonymizer pseudonymizer) {
+
 	}
 
 	@Override
-	public boolean exists(String uuid) {
-		return campaignService.exists(uuid);
+	public List<CampaignDto> getAllAfter(Date date) {
+		return service.getAllAfter(date).stream().map(campaignFormMeta -> toDto(campaignFormMeta)).collect(Collectors.toList());
 	}
 
 	@Override
-	public List<CampaignDto> getAllAfter(Date date) {
-		return campaignService.getAllAfter(date, userService.getCurrentUser())
-			.stream()
-			.map(campaignFormMeta -> toDto(campaignFormMeta))
-			.collect(Collectors.toList());
+	protected void selectDtoFields(CriteriaQuery<CampaignDto> cq, Root<Campaign> root) {
+
 	}
 
 	@Override
 	public List<CampaignDto> getByUuids(List<String> uuids) {
```

Comment:
I think this can be removed as it is present in the super class.

Reply:
campaign is not pseudonymized - for now it makes no sense to add other abstraction for this separation so will implement it